README.md

```diff
@@ -0,0 +1,41 @@
+[![GoDoc](https://godoc.org/github.com/ooni/probe-engine?status.svg)](https://godoc.org/github.com/ooni/probe-engine) [![Build Status](https://travis-ci.org/ooni/probe-engine.svg?branch=master)](https://travis-ci.org/ooni/probe-engine) [![Coverage Status](https://coveralls.io/repos/github/ooni/probe-engine/badge.svg?branch=master)](https://coveralls.io/github/ooni/probe-engine?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/ooni/probe-engine)](https://goreportcard.com/report/github.com/ooni/probe-engine)
+
+# OONI probe measurement engine
+
+This repository contains OONI probe's [measurement engine](
+https://github.com/ooni/spec/tree/master/probe#engine). That is, the
+piece of software that implements OONI nettests.
+
+## API
+
+You can [browse ooni/probe-engine's API](
+https://godoc.org/github.com/ooni/probe-engine?status.svg)
+at [godoc.org](https://godoc.org/). We don't provide any API
+stability guarantees and, as such, we will never release v1.0.0
+of this software.
+
+This repository also allows to build [miniooni](cmd/miniooni), a
+small command line client useful to test the functionality in here
+without integrating with OONI probe. You can browse [the manual
+of this tool](
+https://godoc.org/github.com/ooni/probe-engine/cmd/miniooni)
+at [godoc.org](https://godoc.org/). We may change the command line
```

Comment:
I think this should just link to the miniooni godoc page or maybe just extend the link to include this string too.

Reply:
Same as above, won't link to the godoc homepage